### PR TITLE
Migrate blog from wordpress to github pages

### PR DESCRIPTION
**Pull request by nint8835**
Migrates all existing posts from the old blog, *mostly* intact, so the old blog can be shut down.

### CODE REVIEW

**Comment by jackharrhy** [comment]
Might as well get this merged sooner rather than later to shut down our AWS instance, however I do wish for us to use more aspects of Lektor to automate certain aspects of the blog. (things like `content/blog/contents.lr` should be dynamically created imo).

About to functionally test, but looks good code wise!

**Comment by nint8835** [comment]
Yeah, this was a kinda quick hack so we can shut down the instance. We can improve the system for it later

File: templates/layout.html [hunk 1]
```diff
@@ -70,7 +70,7 @@
                     <a href="https://muncssociety.slack.com/">Slack Community</a>
                 </li>
                 <li class="depth-1">
```
Comment (jackharrhy) [inline templates/layout.html]: Could we sort this above 'Resources'?

File: content/blog/contents.lr [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## November 2017
+
+* [Landing a CS Internship Part 3 - Side Projects](Landing a CS Internship Part 3 - Side Projects)
+
+## December 2017
+
+* [Take Control of Time with Git - Intermediate Git Tutorial](Take Control of Time with Git - Intermediate Git Tutorial)
+
+# January 2018
```
Comment (jackharrhy) [inline content/blog/contents.lr]: This is a header 1, the rest are header 2s

File: content/blog/contents.lr [hunk 3]
```diff
@@ -0,0 +1,28 @@
+title: Blog
+---
+body:
+
+# Blog
+
+Want to add your own blog post? Send us a pull request. For a simple example of how to write a post, refer to [this post's code](https://github.com/MUNComputerScienceSociety/Website/tree/master/content/blog/Welcome everyone!)
+
+## September 2017
```
Comment (jackharrhy) [inline content/blog/contents.lr]: Sort this chronologically starting from latest to oldest?

**Comment by jackharrhy** [comment]
🦍 𝒽𝒶𝓇𝒶𝓂𝒷𝑒 𝒶𝓅𝓅𝓇𝑜𝓋𝑒𝒹

**Pull request merged**